PennMobile/src/main/java/com/pennapps/labs/pennmobile/MainActivity.java

```diff
@@ -272,6 +294,23 @@ public void run() {
         pager.setAdapter(pageAdapter);
     }
 
+    public void addLaundryTabs(LaundryMachineFragment.TabAdapter pageAdapter, final ViewPager pager) {
```

Comment:
instead of creating two identical methods, create a single ```addTabs``` function that accepts parameters of the interface that both TabAdapters extend:
```public void addTabs(FragmentStatePagerAdapter pageAdapter, final ViewPager pager)```